doc/examples/restore_dti.py

```diff
@@ -230,7 +230,7 @@
 closely resembles the baseline distribution of the noiseless signal, and
 demonstrates the utility of the method to data with intermittent
 noise. Importantly, this method assumes that the tensor is a good
-representation of the diffusion signal in the data. If you have reason to
+representation of diffusion signal in data. If you have reason to
```

Comment:
I'd dare to say that the article is necessary/correct.

Reply:
Let the maintainers review it, If wrong then I will change it.

Reply:
The article is necessary.

Reply:
Please, can you update this change @ShrishtiHore?

Otherwise, Both @jhlegarreta and @jchoude are part of the maintainers' team

Reply:
Done. @skoudoro